src/test/java/uk/gov/companieshouse/service/AppealServiceTest.java

```diff
@@ -33,12 +41,27 @@
     private static final String TEST_REASON_TITLE = "This is a title";
     private static final String TEST_REASON_DESCRIPTION = "This is a description";
 
+    private final ObjectMapper mapper = new ObjectMapper();
+    private List<Attachment> testAttachments;
+
     @InjectMocks
     private AppealService appealService;
 
     @Mock
     private AppealRepository appealRepository;
 
+    @Before
+    public void beforeTest() throws IOException {
```

Comment:
:( This is not even run but test still works cause null is still equal null. It does not run cause you used jUnit4 annotation while jUnit5 is our test engine. You should should change it to `BeforeAll`

Reply:
Or `BeforeEeach` to keep variable non static.